requirements/requirements.txt

```
@@ -5,7 +5,7 @@ python-dateutil==2.8.0
 boto3==1.*
 holidays==0.9.*
 matplotlib==3.*
-numpy==1.14.*
+numpy>=1.14.0
```

```suggestion
numpy>=1.14
```

Should we do this in combination with #456?

---

Sure, let's get that one through maybe (somehow I hadn't seen it)